### ISSUE TITLE
Error running httpie: ModuleNotFoundError: No module named 'encodings'

### ISSUE THREAD

**Issue by xbao** [Linuxbrew/homebrew-core#4841]
- [X ] Confirmed this is a problem with `brew install`ing one, specific formula and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Linuxbrew/brew/issues/new.
- [N/A ] If you have a macOS system and can replicate the problem on macOS with Homebrew, please report the issue to [Homebrew/core](https://github.com/Homebrew/homebrew-core) and follow their issue template from your macOS system. If you are unsure, please report the issue to Linuxbrew.
- [X ] Ran `brew update` and retried your prior step?
- [ X] Ran `brew doctor`, fixed all issues and retried your prior step?
 _There's one warning about XDG_DATA_DIRS, but it doesn't seem related_

- [ X] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
 _No logs found_
- [X ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
_Install and run httpie_
- What happened (include command output)
_httpie installed with dependencies just fine. attempting to run httpie gives:_

```
$ http
Fatal Python error: Py_Initialize: Unable to get the locale encoding
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007fde6bcb2700 (most recent call first):
Aborted
```
- What you expected to happen
_The normal output of http_
- Step-by-step reproduction instructions (by running `brew install` commands)
```
$ brew install httpie
$ http
```

config:
```
 brew config
HOMEBREW_VERSION: 1.3.6
ORIGIN: https://github.com/Linuxbrew/brew
HEAD: b39b60a4bfd51e5f2e83f5dc927cdb1f67310686
Last commit: 7 days ago
Core tap ORIGIN: https://github.com/Linuxbrew/homebrew-core
Core tap HEAD: 97922eda510b04a1373b488973880fe569b53c8b
Core tap last commit: 6 hours ago
HOMEBREW_PREFIX: /home/xiao/.linuxbrew
HOMEBREW_REPOSITORY: /home/xiao/.linuxbrew
HOMEBREW_CELLAR: /home/xiao/.linuxbrew/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://linuxbrew.bintray.com
CPU: octa-core 64-bit haswell
Homebrew Ruby: 2.3.3 => /home/xiao/.linuxbrew/Library/Homebrew/vendor/portable-ruby/2.3.3/bin/ruby
Clang: N/A
Git: 1.9.1 => /usr/bin/git
Perl: /usr/bin/perl
Python: /home/xiao/.linuxbrew/opt/python/libexec/bin/python => /home/xiao/.linuxbrew/Cellar/python/2.7.13_1/bin/python2.7
Ruby: /home/xiao/.rubies/ruby-2.4.0/bin/ruby
Java: openjdk version "1.8.0_45-internal"
Kernel: Linux 3.19.0-32-generic x86_64 GNU/Linux
OS: Linux Mint 17.3 Rosa (rosa)
Host glibc: 2.19
/usr/bin/gcc: 4.8.5
glibc: 2.23
gcc: 5.3.0
xorg: N/A
```
doctor:
```
brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Homebrew's share was not found in your XDG_DATA_DIRS but you have
this variable set to include other locations.
Some programs like `vapigen` may not work correctly.
Consider setting the XDG_DATA_DIRS for example like so
    echo 'export XDG_DATA_DIRS="/home/xiao/.linuxbrew/share:$XDG_DATA_DIRS"' >> ~/.bash_profile
```

I'm on quite an old version of Linux Mint. not sure if that's a factor. Happy to do any debugging to help solve this :)

**Comment by xbao** [comment]
I just did an strace, looks like it expects a bunch of things in 'httpie/0.9.9/libexec' that aren't there:
```
readlink("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/bin/python3.6", 0x7ffe342a0610, 4096) = -1 EINVAL (Invalid argument)
open("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/bin/pyvenv.cfg", O_RDONLY) = -1 ENOENT (No such file or directory)
open("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/pyvenv.cfg", O_RDONLY) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/bin/Modules/Setup", 0x7ffe342ab6e0) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/bin/lib/python3.6/os.py", 0x7ffe342a1590) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/bin/lib/python3.6/os.pyc", 0x7ffe342a1590) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/os.py", {st_mode=S_IFREG|0444, st_size=37525, ...}) = 0
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/bin/pybuilddir.txt", 0x7ffe342ab6e0) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/bin/lib/python3.6/lib-dynload", 0x7ffe342ab6e0) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/lib-dynload", 0x7ffe342ab6e0) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/lib/python3.6/lib-dynload", 0x7ffe342ab6e0) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/lib/python3.6/lib-dynload", 0x7ffe342ab6e0) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/lib/python3.6/lib-dynload", 0x7ffe342ab6e0) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/lib/python3.6/lib-dynload", {st_mode=S_IFDIR|0755, st_size=16384, ...}) = 0
...
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python36.zip", 0x7ffe342b2340) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib", {st_mode=S_IFDIR|0755, st_size=4096, ...}) = 0
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python36.zip", 0x7ffe342b1bb0) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6", {st_mode=S_IFDIR|0755, st_size=12288, ...}) = 0
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6", {st_mode=S_IFDIR|0755, st_size=12288, ...}) = 0
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6", {st_mode=S_IFDIR|0755, st_size=12288, ...}) = 0
open("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6", O_RDONLY|O_NONBLOCK|O_DIRECTORY|O_CLOEXEC) = 3
...
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.cpython-36m-x86_64-linux-gnu.so", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.abi3.so", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.so", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.py", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.pyc", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6", {st_mode=S_IFDIR|0755, st_size=12288, ...}) = 0
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.cpython-36m-x86_64-linux-gnu.so", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.abi3.so", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.so", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.py", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings/__init__.pyc", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/Cellar/httpie/0.9.9/libexec/lib/python3.6/encodings", 0x7ffe342b1f30) = -1 ENOENT (No such file or directory)
stat("/home/xiao/.linuxbrew/lib/python3.6/lib-dynload", {st_mode=S_IFDIR|0755, st_size=16384, ...}) = 0
...
write(2, "Fatal Python error: Py_Initializ"..., 69Fatal Python error: Py_Initialize: Unable to get the locale encoding
) = 69
write(2, "ModuleNotFoundError", 19ModuleNotFoundError)     = 19
```

**Comment by xbao** [comment]
I just fixed this by running `easy_install-3.6 httpie`. You can close this issue thanks :)

**Comment by iMichka** [comment]
Sorry we couldn't help you, but you found a workaround. If somebody hits the same problem, a pull request would be great to fix this. Thanks.
